cfrontend/PrintClight.ml

```diff
@@ -116,22 +116,28 @@ let rec print_expr_list p (first, rl) =
 
 (* Statements *)
 
+(* Receiver of call *)
+let print_rec p e =
```

Comment:
The `_rec` suffix means "recursive" to me.  Also, you shoud use the existing `expr` printer with precedence 15. (See below.)

Reply:
More elegant indeed.